projects/rero/ng-core/src/lib/record/editor/type/object-type/object-type.component.html

```diff
@@ -14,12 +14,6 @@
  You should have received a copy of the GNU Affero General Public License
  along with this program.  If not, see <http://www.gnu.org/licenses/>.
 -->
-<!-- main title -->
-<ng-container *ngIf="isRoot()">
-  <legend>
-    <span [tooltip]="to.description">{{ to.label }}</span>
-  </legend>
-</ng-container>
```

Comment:
Are you sure that you can remove this lines?

Reply:
I place this behavior into the editor component. I checked into each resource editor and it seems correct.